src/Airship/Internal/Decision.hs

```diff
@@ -192,7 +192,9 @@ b10 r@Resource{..} = do
     allowed <- lift allowedMethods
     if requestMethod req `elem` allowed
         then b09 r
-        else lift $ halt HTTP.status405
+        else do
+            lift $ addResponseHeader ("Allow", renderHeader allowed)
```

Comment:
`renderHeader` is for HTTP media types, right? What does this end up looking like?